src/cr/cube/matrix/measure.py

```diff
@@ -830,6 +835,80 @@ def _weighted_base_blocks(self):
         return self._second_order_measures.column_weighted_bases.blocks
 
 
+class _AudienceRatio(_ColumnProportions):
+    """Provides the audience-ratio measure for a matrix.
+
+    This measure (also known as "Index") is a 2D np.float64 ndarray of the ratio of the
+    proportions of the first column to the proportions of the second column.
+
+    Audience ratio (Index) is a convenient way of showing the ratio of the Target % and
```

Comment:
This along with smoothing and the reference column statistical significance really makes me wish we had time to design a way to pass arguments into measures better. (so the audience measure would specify which column was the control column instead of being fixed to the second one)

Reply:
yes I do agree, and I had a PR ready for pw signif 2 years ago, but it has been frozen since then because the FE needed to change to much stuff, and since then no PRD has been approved to take care of this issue :(